src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/ChangeLog.md

```diff
@@ -17,5 +17,9 @@
     * Overview of change #1
         - Additional information about change #1
 -->
-## Current Release
- - First release of Get-AzureRmGuestConfigurationPolicy report cmdlet (version: 0.9.0-preview)
+## Current release - version: 0.9.1-preview 
```

Comment:
Just put this data under Current release, and do not update these headings - versions are added by the semantic versioning tool

Reply:
done